### ISSUE TITLE
Getting started guide

### ISSUE THREAD

**Issue by neopostmodern** [kevancress/MeasureIt-ARCH#273]
Just stumbled upon this project and I *think* it'll be really useful for me, but I have a hard time getting started, i.e. rendering anything visible. Therefore I'd like to suggest to add a minimal walkthrough to the docs site.

Things I discovered I needed which I found unintuitive (not bad design, just in need of a hint for new users):
- You need to add lines manually (e.g. "Dynamic Line Group")
- Rendering happens via specific buttons, not Blender rendering
- Most render settings are in the scene tab, not the render tab

Things I haven't discovered yet:
- Setting a white background to actually see something
- Setting a customized drawing style throughout
- Deleting dimensions
- How to make "hidden" edges invisible or dashed
- ~~Add a dimension to one edge, but don't change the rendering/line style of the object~~ Properly and reliably adding dimensions

TL;DR
It would be great to have a walkthrough how to create this picture from a blank project:
![image](https://github.com/kevancress/MeasureIt_ARCH/assets/786840/d1fbca15-94fc-4ae3-9681-04000bc5ff59)

**Comment by skonato** [comment]
It would be great to add something about hatches and section cuts to this list.

**Comment by kevancress** [comment]
@neopostmodern @skonato 

Totally agreed! I've had an empty tutorial section in the documentation for way to long. I'll try and make some time to tackle this soon!

**Comment by neopostmodern** [comment]
Great! Let me know when there is something to proofread/test and I'd be happy to give authentic feedback from a first-time user :)
